Makefile

```diff
@@ -22,6 +22,15 @@ clean clobber build deploy:
 run: build
 	scripts/upload-api
 
+requirements.txt requirements-dev.txt : %.txt : %.txt.in
```

Comment:
Is this Andrey's abstraction around requirements? If so, does this PR also need a requirements.txt.in and requirements-dev.txt.in?

Reply:
Oh snap! This was totally unintentionally... I think I must have accidentally pulled it in from another branch. Deleting!